json/hypercosm.json

```diff
@@ -367,18 +367,29 @@
           "docs": "",
           "kind": {
             "Flags": {
-              "backing": "VU64",
+              "backing": {
+                "Unsigned": "VU64"
```

Comment:
Same thing as with the normal IntType case; no need to clarify signedness. I imagine this is actually generated from the same code.